server/jetstream_cluster.go

```diff
@@ -6491,7 +6490,7 @@ func (mset *stream) processSnapshot(snap *streamSnapshot) (e error) {
 		if gotMsgs || activityInterval == maxActivityInterval {
 			return maxActivityInterval
 		}
-		activityInterval *= 2
+		activityInterval *= 5
```

Comment:
That will limit this whole thing to likely just 1 retry (1sec -> 5sec and then cap of 10sec - as opposed to 1s -> 2s -> 4s -> 8s-> 10s). As discussed before, I would be ok that we remove all that and simply adjust one of the test that was sensitive of the fact that the first catchup request was missed by the leader (before itself was just becoming leader and the sync request subscription was not fully setup).

Reply:
Which test? Do you recall?

Reply:
Yes, it was TestNoRaceJetStreamClusterDirectAccessAllPeersSubs. Maybe just change the timeout here to more than 10 seconds (say 20secs): https://github.com/nats-io/nats-server/blob/d90854a45fe9405198093ab1489f8b3e5e11dcf8/server/norace_test.go#L5318. 
Also this check has been already extended to 20secs (used to be 10 originally I think): https://github.com/nats-io/nats-server/blob/d90854a45fe9405198093ab1489f8b3e5e11dcf8/server/norace_test.go#L5343

Reply:
ok will do that and remove dynamic activityInterval logic.